am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -141,11 +164,13 @@ void whenListOfResourcesButNoReadAccessShouldReturnListOfEnvelopesWithNullDataVa
         List<FilterResourceResponse> expectedResult = ImmutableList.of(
             FilterResourceResponse.builder()
                 .resourceId(resourceId)
+                .relationships(ImmutableSet.of())
                 .data(null)
                 .permissions(createPermissions("", CREATE_PERMISSION))
                 .build(),
             FilterResourceResponse.builder()
                 .resourceId(resourceId + "2")
+                .relationships(ImmutableSet.of())
```

Comment:
ImmutableSet.of() is used lots - could maybe be extracted to a constant called EMPTY_ROLES?